WhiskyMan.Backend/WhiskyMan.Models/BottleDescription/TagReference.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WhiskyMan.Models.BottleDescription
+{
+    public record TagReference
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
```

Comment:
By (my) convention:
Every object which has suffix `Reference` has to have at least field `id` and `name` (may have others if it makes sense e.g. `pictureUrl` but not something too specific e.g. `drunkBottlesLastYear`)